CONFIGURE.md

```diff
@@ -81,11 +85,32 @@ An example excerpt from settings.local.yml:
 ```
 # Default editor
 default_editor: "id"
-# OAuth consumer key for iD
+# OAuth 1 consumer key for iD
 id_key: "8lFmZPsagHV4l3rkAHq0hWY5vV3Ctl3oEFY1aXth"
 ```
 
-Follow the same process for registering and configuring the website/Notes (`oauth_key`), or to save time, simply reuse the same consumer key for each.
+To allow [Notes](https://wiki.openstreetmap.org/wiki/Notes) and changeset discussions to work, follow a similar process, this time registering an OAuth 2 application for the web site:
+
+* Go to "[OAuth 2 applications](http://localhost:3000/oauth2/applications)" on the My settings page.
+* Click on "Register new application".
+* Use Name: "OpenStreetMap Web Site" and Redirect URIs: "http://localhost:3000"
+* Check boxes for the following Permissions
+  * 'Modify the map'
+  * 'Modify notes'
+* On the next page, copy the "Client Secret" and "Client ID"
+* Edit config/settings.local.yml in your rails tree
+* Add the "oauth_application" configuration with the "Client ID" as the value
+* Add the "oauth_key" configuration with the "Client Secret" as the value
```

Comment:
is this really necessary? I don't immediately find anything in the code which uses this setting (b4a1e41968e1befdf6fc14715d95549f5537abca appears to have removed previous occurrences where it was used for oauth1) and the `settings.yml` now only mentions the [`oauth_application`](https://github.com/openstreetmap/openstreetmap-website/blob/73f0be96472a8d021a6347953192ca7b6ec509d7/config/settings.yml#L82-L83) setting. I tried running the website locally without the `oauth_key` setting and it seems to work just fine.